.github/workflows/linux-build.yml

```diff
@@ -44,15 +40,71 @@ on:
       - "scripts/setup-helper-functions.sh"
       - ".github/workflows/linux-build.yml"
       - ".github/workflows/linux-build-base.yml"
-
 permissions:
   contents: read
-
 concurrency:
   group: ${{ github.workflow }}-${{ github.repository }}-${{ github.head_ref || github.sha }}
-  cancel-in-progress: true
-
+  cancel-in-progress: ${{github.ref != 'refs/heads/presto-0.293-clp-connector'}}
 jobs:
   linux-gcc:
     name: Build with GCC
-    uses: ./.github/workflows/linux-build-base.yml
+    runs-on: yscope-gh-runner
+    env:
+      CCACHE_DIR: "${{ github.workspace }}/ccache"
+      USE_CLANG: "${{ 'false' }}"
+    defaults:
+      run:
+        shell: bash
+        working-directory: velox
+    steps:
+      - name: Node Setup and Cleanup
+        working-directory: ${{ github.workspace }}
+        run: |
+          df -h
+          rm -rf /usr/local/lib/android
+          rm -rf /usr/share/dotnet
+          rm -rf /usr/share/dotnet
+          rm -rf /opt/ghc
+          rm -rf /opt/hostedtoolcache/CodeQL
+          rm -rf "/usr/local/share/boost"
+          rm -rf "$AGENT_TOOLSDIRECTORY"
+          docker image prune --all --force
+          docker builder prune -a
```

Comment:
Where's this from and do we still need it now that we've reduced the size of the build?